### PR TITLE
Milestone: Works and About self section

### PR DESCRIPTION
**Pull request by mpa-mxiang**
In this section, I:
- Added sections for works and about myself in the index.html
- Added elements style for new sections in the style.css
- Added related images in the img folder

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -96,3 +96,138 @@ a {
...
+  font-size: 1.2rem;
+}
+
+.card button {
+  color: #396df2;
+  border: 2px solid #396df2;
+  background-color: white;
+  border-radius: 8px;
+  padding: 1rem;
+  flex: none;
```
Comment (Kamba56) [inline style.css:172]: - [x] I noticed the borders on your buttons are thicker than the ones in Figma template. Kindly fix this, please. You are expected to stick to design as much as possible.
### Your design 

### Figma template
f1

- [x] Kindly add a hover and pressed state to all your buttons to match the figma template. You can use the `:hover` and `:active` pseudo-classes.
btn1

File: index.html [hunk 2]
```diff
@@ -37,10 +37,119 @@ <h1>I'm Jiachun, Glad to see you</h1>
...
+            <p class="description">
+                A daily selection of privately personalized reads. No accounts or 
+                sign-ups required.
+            </p>
+            <ul class="technologies">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <div>
```
Comment (Kamba56) [inline index.html]: - [x] Please use an indentation of two spaces in all your documents. This is a convention and also best practice.

File: index.html [hunk 3]
```diff
@@ -37,10 +37,119 @@ <h1>I'm Jiachun, Glad to see you</h1>
...
+                </div>
+                <div class="language">
+                    <img class="language-img" src="./img/html.jpg" alt="JavaScript image">
+                    <p>HTML</p>
+                </div>
+                <div class="language">
+                    <img class="language-img" src="./img/css.jpg" alt="JavaScript image">
+                    <p>CSS</p>
+                </div>
+
```
Comment (Kamba56) [inline index.html]: - [x] I noticed that your about me section is not complete, kindly complete it, please.
### Your design

### Figma
f2

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @mpa-mxiang , 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @mpa-mxiang 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request

You have submitted a project, that is failing linter checks

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by victorokoroji — APPROVED** [review]
Hi @mpa-mxiang, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- Good UI design ✅ 
- Linters are working properly ✅ 


## Optional suggestions
- I will strongly recommend that you always write a descriptive branch name describing changes that you introduced or the feature you are working on. Your branch name currently doesn't represent what you are currently working on there making your repo unprofessional. Kindly improve on this because its a reason for invalidation of your project 👍🏽 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**